cli/shared/src/main/scala/ch/epfl/bluebrain/nexus/cli/types/EventEnvelope.scala

```diff
@@ -1,6 +1,6 @@
 package ch.epfl.bluebrain.nexus.cli.types
 
 /**
- * Event wrapper adding the offset metadata.
+  * Event wrapper adding the offset metadata.
  **/
-final case class EventEnvelope(offset: Offset, event: Event)
+final case class EventEnvelope(offsetOpt: Option[Offset], event: Event)
```

Comment:
is there a case where an event wouldn't have an offset?

Reply:
In our case probably not, but the standard, the API and the `ServerSent` model of `fs2` all allow for this field to be optional.

Reply:
I'm happy to merge this as is, but we should be able to rely on having an offset. And fail on the first event that doesn't have one. It's part of the api.